controllers/consoleplugin/consoleplugin_objects.go

```diff
@@ -397,6 +404,19 @@ func (b *builder) setFrontendConfig(fconf *config.FrontendConfig) error {
 		Mark:  dedupJustMark,
 		Merge: dedupMerge,
 	}
+	if b.desired.Processor.MultiClusterDeployment != nil && *b.desired.Processor.MultiClusterDeployment {
+		fconf.Features = append(fconf.Features, "multiCluster")
+	} else {
+		skipColumns = append(skipColumns, "cluster")
+	}
+	if b.desired.Processor.AddZone != nil && *b.desired.Processor.AddZone {
+		fconf.Features = append(fconf.Features, "zones")
+	} else {
+		skipColumns = append(skipColumns, "zone")
+	}
```

Comment:
we could also add to columns config something like a `requiresFeature` field, that could be managed either here or on the frontend, to avoid having specific code for each feature here to skip columns.
E.g:

```yaml
  - id: ClusterName
    name: Cluster
    tooltip: The cluster ID or Name.
    field: K8S_ClusterName
    filter: cluster_name
    default: false
    width: 15
    requiresFeature: multiCluster
```

Reply:
Yes I'm fine with that